main/options/options.cc

```diff
@@ -366,8 +366,8 @@ buildOptions(const vector<pipeline::semantic_extension::SemanticExtensionProvide
     options.add_options("advanced")("enable-experimental-requires-ancestor",
                                     "Enable experimental `requires_ancestor` annotation");
 
-    options.add_options("advanced")("enable-experimental-lsp-extract-to-variable",
-                                    "Enable experimental LSP feature: Extract To Variable");
+    options.add_options("advanced")("enable-beta-lsp-extract-to-variable",
+                                    "Enable beta LSP feature: Extract To Variable");
```

Comment:
When we move a feature to beta, we don't rename the option for the specific feature. So this can remain `enable-experimental-*`.